kasa/cli.py

```diff
@@ -818,13 +818,13 @@ async def emeter(dev: Device, index: int, name: str, year, month, erase):
         else:
             emeter_status = dev.emeter_realtime
 
-        echo("Current: %s A" % emeter_status["current"])
-        echo("Voltage: %s V" % emeter_status["voltage"])
-        echo("Power: %s W" % emeter_status["power"])
-        echo("Total consumption: %s kWh" % emeter_status["total"])
+        echo("Current: {} A".format(emeter_status["current"]))
+        echo("Voltage: {} V".format(emeter_status["voltage"]))
+        echo("Power: {} W".format(emeter_status["power"]))
+        echo("Total consumption: {} kWh".format(emeter_status["total"]))
```

Comment:
Maybe for consistency these should be f strings with `.get()`

Reply:
I'll remove the `--fix-unsafe` commit, we can revisit this in a separate PR (as we may want to add some more ruff checks to the mix, too).